### ISSUE TITLE
Creating Additional Calendars In "My calendars"

### ISSUE THREAD

**Issue by Ledhed152** [derekantrican/GAS-ICS-Sync#124]
I have only one calendar linked under sourceCalendars (Code.gs, line 19).

The calendar syncs correctly, but also creates 2 separate calendars under "My calendars" named "targetCalendar1" and "targetCalendar2". Neither of these calendars contain any information and can be deleted, but are created again upon next sync (var howFrequent = 15).

![GAS Screen 1](https://user-images.githubusercontent.com/64606637/80743414-86617f80-8ae2-11ea-89f9-03c11ea6211a.jpg)

**Comment by derekantrican** [comment]
`targetCalendar1` and `targetCalendar2` are the name of the target calendars in the code when first downloaded: https://github.com/derekantrican/GAS-ICS-Sync/blob/62fec0a1b50af0178901ef06f13d5f6e4b278167/Code.gs#L21 Could you check to make sure they don't exist somewhere else in your code by doing a search (Ctrl+F) for those names?

**Comment by derekantrican** [comment]
You can also check to make sure you don't have some other copy of the script hanging around in your Google Drive that is running

**Comment by Ledhed152** [comment]
I did a Ctrl+F search for both and it returned no results.

I did have another copy in my Google Drive, I have since deleted that copy and waited for a re-sync and it's still creating the extra calendars.

**Comment by derekantrican** [comment]
You can also try searching in Google Drive for the same calendar names and it will search the contents of your documents (see below):

![image](https://user-images.githubusercontent.com/1558019/80752177-ea367900-8adf-11ea-8c52-51569b87aae0.png)

**Comment by Ledhed152** [comment]
As suggested, I searched my Google Drive for both names, it only returned results that were in the Trash folder. I permanently deleted those files from the trash.

Also, there was a calendar in "Other calendars" that I had synced (the normal Google Calendar way) to the same source with the same name that I had specified in the script. I changed the name of that so they longer match.

Since permanently deleting the files in the Google Drive trash and changing the name of that calendar, I have had no issues.

Thank you for making this, and taking the time to respond to issues. It works great!
